app/src/main/java/org/oppia/android/app/administratorcontrols/LogoutDialogFragment.kt

```diff
@@ -0,0 +1,25 @@
+package org.oppia.android.app.administratorcontrols
+
+import android.app.Dialog
+import android.os.Bundle
+import androidx.appcompat.app.AlertDialog
+import androidx.fragment.app.DialogFragment
+import org.oppia.android.R
+import org.oppia.android.app.profile.ProfileChooserActivity
+
+/** [DialogFragment] that gives option to either cancel or logout from current profile. */
+class LogoutDialogFragment : DialogFragment() {
+
```

Comment:
Not sure, but shouldn't the letter 'o' be uppercase?

@rt4914 @ArpitShukIa WDYT?

Reply:
Which 'o'?

Reply:
Yes, I meant LogOutDialogFragment but I am not really sure if we want to change it or not.

Reply:
The noun form is correct here (i.e. this is the dialog where logouts occur). The documentation should say 'log out' since the verb form is the action the user is taking. Another way to check: methods almost certainly should be logOut since they are verb phrases, and classes will usually be logout since they are noun phrases.

Reply:
Sounds good, thanks for the clarification @BenHenning.
This can be resolved now.

Reply:
> The documentation should say 'log out' since the verb form is the action the user is taking

Fixed it.